app/src/lib/feature-flag.ts

```diff
@@ -148,3 +148,10 @@ export function enableDefaultBranchSetting(): boolean {
 export function enableUnhandledRejectionReporting(): boolean {
   return enableBetaFeatures()
 }
+
+/**
+ * Should we allow cherry picking
+ */
+export function enableCherryPicking(): boolean {
+  return false // enableBetaFeatures()
```

Comment:
You probably already know, but FYI you can leave it enabled for development only using `enableDevelopmentFeatures()` if you don't want it to get to beta yet.

Reply:
Thank you. Right now this just has a menu item that does nothing so I figured to leave it disabled until it invokes the cherry picking branch modal (next PR).. 😄